scripts/run.sh

```diff
@@ -30,33 +27,35 @@ source "$SUBNET_EVM_PATH"/scripts/versions.sh
 source "$SUBNET_EVM_PATH"/scripts/constants.sh
 
 VERSION=$avalanche_version
+
+# "ewoq" key, do not include "0x"
 DEFAULT_ACCOUNT="0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC"
 GENESIS_ADDRESS=${GENESIS_ADDRESS-$DEFAULT_ACCOUNT}
 
-SKIP_NETWORK_RUNNER_START=${SKIP_NETWORK_RUNNER_START:-false}
-SKIP_NETWORK_RUNNER_SHUTDOWN=${SKIP_NETWORK_RUNNER_SHUTDOWN:-false}
+SKIP_NETWORK_RUNNER_SHUTDOWN=${SKIP_NETWORK_RUNNER_SHUTDOWN:-true}
 RUN_SIMULATOR=${RUN_SIMULATOR:-false}
-ENABLE_SOLIDITY_TESTS=${ENABLE_SOLIDITY_TESTS:-false}
-AVALANCHE_LOG_LEVEL=${AVALANCHE_LOG_LEVEL:-INFO}
+AVALANCHE_LOG_LEVEL=${AVALANCHE_LOG_LEVEL:-WARN}
 ANR_VERSION=$network_runner_version
 GINKGO_VERSION=$ginkgo_version
-GINKGO_FOCUS_FLAGS=${GINKGO_FOCUS_FLAGS}
-GINKGO_SKIP_FLAGS="\[Precompiles\]"
-if [[ ${ENABLE_SOLIDITY_TESTS} == true ]]; then
-  GINKGO_SKIP_FLAGS=""
+
+# by default, "run.sh" should not run any tests...
+# simulator tests are not implemented in ginkgo
+# it instead runs external binary "simulator"
+# so we exclude all e2e tests here
+# ref. https://onsi.github.io/ginkgo/#spec-labels
+GINKGO_LABEL_FILTER="!precompile-upgrade && !solidity-counter"
```

Comment:
label selector is more straightforward to understand, so we can deprecate filter flags.